securedrop/journalist_templates/admin_add_user.html

```diff
@@ -6,17 +6,39 @@
 
 <form method="post">
   <input name="csrf_token" type="hidden" value="{{ csrf_token() }}">
-  <input name="password" type="hidden" value="{{ password }}">
-  <p><input name="username" type="text" placeholder="{{ gettext('Username') }}"></p>
+  <p>
+    {{ form.password(value=password, id=False) }}
+    <br>
+    {% for error in form.password.errors %}
+      <span class="form-validation-error">{{ error }}</span>
+    {% endfor %}
+  </p>
+
+  <p>
+    {{ form.username(placeholder=gettext('Username')) }}
+    <br>
+    {% for error in form.username.errors %}
+      <span class="form-validation-error">{{ error }}</span>
+    {% endfor %}
+  </p>
+
   <p>{{ gettext("The user's password will be:") }} <span class="password" id="password">{{ password }}</span></p>
   <p>
-    <input name="is_admin" id="is-admin" type="checkbox">
+    {{ form.is_admin(id="is-admin") }}
     <label for="is_admin">{{ gettext('Is Administrator') }}</label>
+    <br>
+    {% for error in form.is_admin.errors %}
+      <span class="form-validation-error">{{ error }}</span>
+    {% endfor %}
   </p>
   <p>
-    <input name="is_hotp" id="is-hotp" type="checkbox">
+    {{ form.is_hotp(id="is-hotp") }}
     <label for="is_hotp">{{ gettext("Is using a YubiKey [HOTP]") }} </label>
-    <input name="otp_secret" type="text" placeholder="{{ gettext('HOTP Secret') }}" size="60"><br>
+    {{ form.otp_secret(placeholder=gettext('HOTP Secret'), size="60") }}
+    <br>
+    {% for error in form.is_hotp.errors + form.otp_secret.errors %}
+      <span class="form-validation-error">{{ error }}</span>
```

Comment:
It looks like the display of multiple errors could be confusing because they will show on the same line. Maybe the **<br>** should be in the **for** loop ?

Reply:
Multiple errors will render like this now: 
![screen shot 2017-11-10 at 10 32 43 pm](https://user-images.githubusercontent.com/7832803/32687203-5e998b60-c66c-11e7-9d62-42a983c339cd.png)